scripts/buildCSS.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+
+const CSSInterface = require('react-with-styles-interface-css').default;
```

Comment:
wait; this is a bug in the css interface. There should never be a `default` property on `module.exports` for a package's entry point :-/

It's possible your addition of `registerMaxSpecificity` caused this; if so, maybe we should remove it and have it be under a different entry point (`require('react-with-styles-interface-css/registerMaxSpecificity')`)

Reply:
https://github.com/airbnb/react-with-styles-interface-css/pull/8 should fix some things!